R/utilities-simulation-results.R

```diff
@@ -34,15 +34,23 @@ getOutputValues <- function(simulationResults,
   quantitiesOrPaths <- quantitiesOrPaths %||% simulationResults$allQuantityPaths
   quantitiesOrPaths <- c(quantitiesOrPaths)
 
-  # If quantities are passed, get their paths.
-  paths <- quantitiesOrPaths
-  if (isOfType(paths, Quantity)) {
-    paths <- unlist(lapply(paths, function(x) x$path))
+  if (length(quantitiesOrPaths) == 0) {
+    return(list(data = NULL, metaData = NULL))
   }
-  paths <- unique(paths)
 
-  if (length(paths) == 0) {
-    return(list(data = NULL, metaData = NULL))
+  # If quantities are passed, get their paths.
+  if (isOfType(quantitiesOrPaths, Quantity)) {
+    quantities <- uniqueEntities(quantitiesOrPaths)
+    paths <- unlist(

```

Comment:
unless the formatter does it autimatically (!?!) put this in one line. (unlist(lapply())

Reply:
Re-run styler on the package.